official/resnet/imagenet.py

```diff
@@ -83,11 +83,11 @@ def create_readable_names_for_imagenet_labels():
   (since 0 is reserved for the background class).
 
   Code is based on
-  https://github.com/tensorflow/models/blob/master/inception/inception/data/build_imagenet_data.py#L463
+  https://github.com/tensorflow/models/blob/master/research/inception/inception/data/build_imagenet_data.py
```

Comment:
This is deprecated and should instead point to https://github.com/tensorflow/models/blob/master/research/slim/datasets/build_imagenet_data.py

Reply:
Thanks Jon! Just set up a separate pull request for this: https://github.com/tensorflow/models/pull/2455